dist/common/scripts/scylla_util.py

```diff
@@ -296,13 +298,17 @@ def firstNvmeSize(self):
         """return the size of first non root NVME disk in GB"""
         if self.__firstNvmeSize is None:
             ephemeral_disks = self.getEphemeralOsDisks()
-            firstDisk = ephemeral_disks[0]
-            firstDiskSize = self.get_file_size_by_seek(os.path.join("/dev/", firstDisk))
-            firstDiskSizeGB = firstDiskSize/1024/1024/1024
-            if firstDiskSizeGB >= self.GCP_NVME_DISK_SIZE_2020:
-                self.__firstNvmeSize = firstDiskSizeGB
+            if len(ephemeral_disks) > 0:
```

Comment:
Is this change part of the same issue or a different bug?

Reply:
same issue, if no nvmes found, we shall still continue and not fail